### PR TITLE
Clarify treatment of zero weights

### PR DESCRIPTION
**Pull request by amdw**

### CODE REVIEW

File: content/en/templates/taxonomy-templates.md [hunk 1]
```diff
@@ -158,7 +158,11 @@ Hugo uses both `date` and `weight` to order content within taxonomies.
 
 Each piece of content in Hugo can optionally be assigned a date. It can also be assigned a weight for each taxonomy it is assigned to.
 
-When iterating over content within taxonomies, the default sort is the same as that used for [section and list pages]() first by weight then by date. This means that if the weights for two pieces of content are the same, than the more recent content will be displayed first. The default weight for any piece of content is 0.
+When iterating over content within taxonomies, the default sort is the same as that used for [section and list pages]() first by weight then by date. This means that if the weights for two pieces of content are the same, than the more recent content will be displayed first.
```
Comment (coliff) [inline content/en/templates/taxonomy-templates.md:161]: I think:
than the more recent content will be displayed first
should be:
**then** the more recent content will be displayed first

Comment (amdw) [reply content/en/templates/taxonomy-templates.md]: Agreed! That problem was there in the old version too, but I'm happy to fix it here. Done!

Comment (coliff) [reply content/en/templates/taxonomy-templates.md]: Cool. I can't review this from a technical perspective (I don't know the logic), but It looks good I think - though it looks like that  `[section and list pages]()` is an error (empty link).
Again - not your fault as it was there before, but maybe we should fix in this same PR. :-)

Comment (amdw) [reply content/en/templates/taxonomy-templates.md]: Thanks. I'm not sure what page that link was supposed to refer to, so I've removed the link altogether.

For the record, the zero-weight special-casing logic I'm referring to here is:

https://github.com/gohugoio/hugo/blob/cdfd1c99baa22d69e865294dfcd783811f96c880/resources/page/pages_sort.go#L113

**Review by coliff — APPROVED** [review]

**Pull request merged**